sqls/subjects/subjectWithHashExists.sql

```diff
@@ -0,0 +1,33 @@
+IF EXISTS(
+    SELECT *
+    FROM sysobjects
+    WHERE type = 'P' AND name = 'subjectWithHashExists'
+)
+BEGIN
+    DROP PROCEDURE subjectWithHashExists
+END
+GO
+
+CREATE PROCEDURE [dbo].subjectWithHashExists(@hash VARCHAR(40)) AS
```

Comment:
the DB field is defined as CHAR(40) but stored procedure argument is VARCHAR(40)